### PR TITLE
[Do not merge] AKS Agentpool API change

### PR DESCRIPTION
**Pull request by xizha162**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json [hunk 1]
```diff
@@ -0,0 +1,1929 @@
...
+          "x-ms-enum": {
+            "name": "NetworkPlugin",
+            "modelAsString": true
+          },
+          "description": "Network plugin used for building Kubernetes network."
+        },
+        "networkPolicy": {
+          "type": "string",
+          "enum": [
+            "calico"
```
Comment (feiskyer) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json]: missing azure

**Comment by feiskyer** [comment]
Also missed `apiServerAuthorizedIPRanges`?

**Comment by sarangan12** [comment]
@xizhamsft  

1. Since this still has Do Not Merge in the header
2. No movement in this for the past 1 month 
3. Conflicts in the files

Do you still want this PR to be opened? Do you want me to review or close this pr?

**Comment by sarangan12** [comment]
Closing this PR for now. If you want this reviewed resolve the conflicts and reopen it again. Thanks

**Pull request closed without merging**